pages/linux/netselect-apt.md

```diff
@@ -0,0 +1,20 @@
+# netselect-apt
+
+> Create sources.list for Debian mirror with the lowest latency.
+> More information: <https://manpages.debian.org/buster/netselect-apt/netselect-apt.1.en.html>.
+
+- Create sources.list for the lowest latency server:
+
+`sudo netselect-apt`
+
+- Specify Debian branch, stable is used by default:
+
+`sudo netselect-apt {{testing}}`
+
+- Include non-free section:
+
+`sudo netselect-apt -n`
```

Comment:
Is there a long-form of this argument? We generally prefer long form arguments in tldr pages.

Reply:
Yes, there's `--non-free` for this. I'll make the change.

Reply:
Done.

Reply:
:+1: